### DISCUSSION TITLE
그 외 논의 사항

### DISCUSSION THREAD

**Discussion by locodingve** [prgrms-web-devcourse/slam-server#130] [General]
그 외 논의 사항

**Comment by locodingve** [comment]
백앤드 스크럼 일정과 스크럼 방식?을 논의해봐야 할 것 같습니다. 

- 스크럼 일정 논의 : 취업한 사람한 사람이 생겨서, 지금의 스크럼 일정을 변경할 필요성이 있습니다. 
- 스크럼 방식 논의 : 월 단위로 수행할 업무 설정, 주 단위로 수행할 업무 설정, 그리고 스크럼 때 진행상황 자세하게 공유하기 (업무 진행시 루즈해 질 수 있음을 방지하기 위해)  

본 논의는 슬랙으로 할 예정입니다. 
평일날 제가 슬랙으로 공유드리겠습니다. 
메모 차원으로 작성한 내용입니다.

**Comment by locodingve** [comment]
설정 파일을 문서화 하였으면 합니다. 

현재 설정 파일의 버전 관리는 잘 되지 않고 있습니다. 
그래서 구글 드라이브를 통해 문서 관리를 하고자 합니다. 
민감한 정보가 담겨있기 때문에, 팀원들끼리 공유가 될 예정입니다. 
그리고 백앤드의 설정 파일임으로 젤리, 헤이 그리고 저 는 수정 및 읽기 권한을 드리며, 
프론트 개발자 분들께는 읽기 권한만 드릴 예정입니다. 

![image](https://user-images.githubusercontent.com/88185304/152508768-64b4164d-d5e6-4900-83a4-4e4fd13ce7c9.png)

위의 이미지는 버전관리가 되고 있다는 것을 보여줍니다. 
평일날 한 번 더 슬랙으로 공유드리겠습니다.

**Reply by KwonYeKyeong** [reply]
구글문서를 통해서 설정파일 관리하는 거 너무 좋은 의견인 거 같아요!! 쌍따봉!!👍👍
